Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup Label="Package Versions">
+    <NpgsqlVersion>4.0.3</NpgsqlVersion>
+    <NpgsqlNodaTimeVersion>1.0.0</NpgsqlNodaTimeVersion>
```

Comment:
I'm not sure I see the value of externalizing dependency versions of packages like NodaTime and NetTopologySuite, which are only used by one project each - it seems to make more sense just to leave those versions in their respective project csprojs... This is unlike, say, `EFCoreVersion` which is used in many projects and so there's real value in externalizing it.

Also, I think we should strive to have hotfix branches track our vNext branch (dev) as much as possible. In other words, if we do decide to bring out NodaTime and NetTopologySuite versions out of their projects, this needs to be done in dev as well.

Reply:
That sounds good. I'll move those version back into their files.

Reply:
That... makes sense. Maybe do the same in dev then?

Reply:
Sure, I'll submit a PR shortly.